### PR TITLE
Android Employee App Intial - Spring 1

### PR DESCRIPTION
**Pull request by ditchfieldcaleb**
Employee App now working. Pulls Times from server using Employee with ID 0, and submits times as well.
Has some amount of verification for times and error checking.

### CODE REVIEW

File: AndroidEmployeeApp/app/src/main/java/edu/gatech/ledpathways/androidemployeeapp/http/AsyncRequestTimes.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        List<Timeblock> list = null;
+        try {
+            URLConnection connect =  params[0].openConnection();
+            list = ParseJsonTimeblock.readJsonStream(connect.getInputStream());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        SystemClock.sleep(700);
+        return list;
```
Comment (mmccoy37) [inline AndroidEmployeeApp/app/src/main/java/edu/gatech/ledpathways/androidemployeeapp/http/AsyncRequestTimes.java]: return an empty list if the try/catch fails. The other code in the main activity and onPostExecute() doesnt check for null.

File: AndroidEmployeeApp/app/src/main/java/edu/gatech/ledpathways/androidemployeeapp/http/AsyncSendTimes.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+
+
+            //client.setRequestProperty("User-Agent", "Mozilla/5.0 (Macintosh; U; PPC; en-US; rv:1.3.1)");
+            //client.setRequestProperty("Accept-Charset", "UTF-8");
+            //client.setRequestProperty("Content-Language", "en-US");
+            //client.setUseCaches(false);
+            //client.setDoInput(true);
+            //client.setDoOutput(true);
+
```
Comment (mmccoy37) [inline AndroidEmployeeApp/app/src/main/java/edu/gatech/ledpathways/androidemployeeapp/http/AsyncSendTimes.java]: if the doInBackground() code works remove the commented out code to clean this up. (I'll do the same lol)

**Comment by mmccoy37** [comment]
Largest thing is /android-guest-app/* should not be pushed. it is old code. only push the folder with the androidEmployeeApp otherwise it will overwrite my latest changes with old stuff.

**Comment by ditchfieldcaleb** [comment]
Closing pull request, using fix-employee-app branch instead.

**Pull request closed without merging**